HesoyamHospital/GraphicEditor/Repository/EquipmentAndMedicineRepository.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace GraphicEditor
```

Comment:
Same thing here. Repository should be within a Backend project.
We want to use the same model and repository to be used from every application, maintaining consistency.